contrib/msggen/msggen/gen/grpc2py.py

```diff
@@ -87,6 +88,10 @@ def amount2msat(a):
             return a.msat
 
 
+        def amount2sat(a):
+            return a.msat / 1000
```

Comment:
```suggestion
            return a.msat // 1000
```
Do we want integer division here?  Any remainder after division should indicate a problem, but if we're only using it to directly convert types it shouldn't theoretically be an issue.

Reply:
I assumed an `Amount` already is a correctly parsed value and this should be fine, otherwise we should raise an error here, no?